include/envoy/network/udp_packet_writer_handler.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+
+#include "envoy/api/io_error.h"
+#include "envoy/buffer/buffer.h"
+#include "envoy/network/address.h"
+#include "envoy/network/socket.h"
+#include "envoy/stats/scope.h"
+#include "envoy/stats/stats_macros.h"
+
+namespace Envoy {
+namespace Network {
+
+/**
+ * Udp Default Packet Writer Name
+ **/
+static const std::string DefaultWriterName{"udp_default_writer"};
```

Comment:
static doesn't mean anything in header file.
use `constexpr std::string_view` instead of `const std::string`.

Reply:
Thanks for pointing it out Lizan, I have dropped the `static` keyword here and converted `std::string` to `absl::string_view`